.github/workflows/mkosi.yml

```diff
@@ -29,11 +29,14 @@ jobs:
     - name: Install
       run: sudo apt-get update && sudo apt-get install --no-install-recommends python3-pexpect
 
+    - name: Symlink
+      run: ln -s .mkosi/mkosi.${{ matrix.distro }} mkosi.default
```

Comment:
-sf ? just in case

Reply:
I wouldn't, we don't expect a mkosi.default file to be there in CI. We should fail if one appears so we know that we're overwriting something when we symlink to mkosi.default.